### ISSUE TITLE
schema returned by Neo4jGraphQL lead to subscription error

### ISSUE THREAD

**Issue by juan2lv** [neo4j/graphql#886]
**Describe the bug**
A simple subscription test like this:
```
// Schema definition
  const typeDefs = gql`
    type Query {
      currentNumber(value: Int): Int
    }

    type Subscription {
      numberIncremented(value: Int): Int
    }
  `;

  // Resolver map
  const resolvers = {
    Query: {
      currentNumber(_source, {value}) {
        pubsub.publish("NUMBER_INCREMENTED", { numberIncremented: value });
        return value;
      },
    },
    Subscription: {
      numberIncremented: {
        // subscribe: () => pubsub.asyncIterator(["NUMBER_INCREMENTED"]),
        subscribe: withFilter(
          () => pubsub.asyncIterator('NUMBER_INCREMENTED'),
          (payload, variables) => {
            console.log(payload);
            console.log(variables);
            return payload.numberIncremented % variables.value === 0;
          },
        ),
      },
    },
  };

  const schema = makeExecutableSchema({ typeDefs, resolvers });
  // neo4j setup
  const neoSchema = new Neo4jGraphQL({
    typeDefs,
    driver,
    resolvers,
    config: {
      enableRegex: true,
      jwt: {
          secret: process.env.JWT_SECRET,
      },
    },
  });
```
If I pass `neoSchema` to `SubscriptionServer.create()`, I will get error when subscription about to receive, if I pass `variables` in when subscribe:
```
{
  "error": {},
  "errors": [
    {
      "message": "Cannot read property 'args' of undefined",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "numberIncremented"
      ]
    }
  ]
}
```
If I use `schema` which returned by `makeExecutableSchema()`, or use `neoSchem` but no `variables` when subscribe, everything works fine.

**System (please complete the following information):**

* Version: "@neo4j/graphql": "^2.3.0"
* Node.js version: v14.18.1

**Comment by neo4j-team-graphql** [comment]
Many thanks for raising this bug report @juan2lv. :bug: We will now attempt to reproduce the bug based on the steps you have provided.

Please ensure that you've provided the necessary information for a minimal reproduction, including but not limited to:
* Type definitions
* Resolvers
* Query and/or Mutation (or multiple) needed to reproduce

If you have a support agreement with Neo4j, please link this GitHub issue to a new or existing Zendesk ticket.

Thanks again! :pray:

**Comment by darrellwarde** [comment]
I could have sworn that this was something we fixed - can ensure that you are running the latest 2.5.6 and check again?

**Comment by juan2lv** [comment]
> I could have sworn that this was something we fixed - can ensure that you are running the latest 2.5.6 and check again?

It's fixed in 2.5.6, thanks!
